tests/test_db_crawler.py

```diff
@@ -53,24 +52,77 @@ def test_fetch_crawl_row_by_postgresql_url(self):
+                "embedding_model": "test-model"
+            }
+            crawler.store_embedding_item(cursor, item)
+            self.connection.rollback()
+        self.assertEqual(item["token_id"], "be612259-9b52-42fd-8d0b-d72120efa3b6")
+
+    def test_fetch_crawl_ids_without_chunk(self):
+        """Test fetching crawl IDs without a chunk."""
+        with db.cursor(self.connection) as cursor:
+            id = crawler.fetch_crawl_ids_without_chunk(cursor)
+            self.connection.rollback()
+        self.assertEqual(id, [])
```

Comment:
This assertion isn't adequately testing `fetch_crawl_ids_without_chunk`. A better test would be to first populate the table with a mix of crawls, some with chunk IDs and some without. Then, we should verify that the function's results match exactly the crawls that don't have chunk IDs, before performing a rollback.